src/physics/site_types/qubit.jl

```diff
@@ -175,13 +175,16 @@ function ITensors.op(::OpName"Ry", ::SiteType"Qubit"; θ::Number)
 end
 
 # Rotation around Z-axis
-function ITensors.op(::OpName"Rz", ::SiteType"Qubit"; θ::Number)
+function ITensors.op(::OpName"Rz", ::SiteType"Qubit"; θ=nothing, ϕ=nothing)
+  isone(count(isnothing, (θ, ϕ))) || error(
+    "Must specify the keyword argument `θ` (or the deprecated `ϕ`) when creating an Rz gate, but not both.",
+  )
+  isnothing(ϕ) && (ϕ = θ)
```

Comment:
I think this should have the opposite logic, i.e.
```julia
isnothing(θ) && (θ = ϕ)
```
and then use `θ` since `ϕ` is the deprecated one.

Reply:
sure, can swap